app/src/forms/form/service.js

```diff
@@ -369,6 +369,16 @@ const service = {
 
       await trx.commit();
 
+      const { subscribe } = await service.readForm(formId);
+      if (subscribe && subscribe.enabled) {
+        const subscribeConfig = await service.readFormSubscriptionDetails(formId);
+        const config = Object.assign({}, subscribe, subscribeConfig);
+        const formVersion = new FormVersion();
+        formVersion.id = formVersionId;
+        formVersion.formId = formId;
+        service.postSubscriptionEvent(config, formVersion, null, SubscriptionEvent.FORM_PUBLISHED);
+      }
```

Comment:
I'm getting a "publish" event when a version is unpublished. The publish boolean created on line 353 could be used to restrict this to only the publishing of new versions. Can you wrap this whole chunk of new code in an `if (publish) { ... }`?

Reply:
This is all good with the code fixes.